### ISSUE TITLE
Rubberhose animation pinned to top left corner when exported

### ISSUE THREAD

**Issue by grodasy10** [airbnb/lottie-web#1682]
When I preview my animation the rubber hose layers are pinned to the top left corner of the animation. Otherwise the animation is working fine.
Assets attached, and can preview here: https://lottiefiles.com/share/Xu9fVf
[Walking dog.zip](https://github.com/airbnb/lottie-web/files/3304484/Walking.dog.zip)

**Comment by Inventsable** [comment]
See #1676

**Comment by grodasy10** [comment]
Thanks, but that doesn't seem to be the problem here. 
I have 'hidden' and 'guides turned on.
<img width="896" alt="Screen Shot 2019-06-19 at 3 29 14 pm" src="https://user-images.githubusercontent.com/51983532/59739169-1de12980-92a7-11e9-9dc9-87dc648c3e3c.png">
<img width="897" alt="Screen Shot 2019-06-19 at 3 29 05 pm" src="https://user-images.githubusercontent.com/51983532/59739170-1f125680-92a7-11e9-9786-22d544a6abd5.png">

**Comment by Inventsable** [comment]
Apologies for jumping the gun. I looked at your file, but something feels strange about the setup:

<img width="1051" alt="ice_screenshot_20190618-224246" src="https://user-images.githubusercontent.com/37279677/59739683-79190a80-921a-11e9-883a-660de038325f.png">

I parented all the controllers to a null, then shifted it's position and scaling. In doing so, noticed that you had other Nulls in your Solids folder which I don't see here, and that your Body-top, Floor and BG are scaled up to 500%. Did you previously have two Nulls that you used to change the positioning or scaling of the Rubberhose limbs?

**Comment by grodasy10** [comment]
Thanks for the reply, I didn't actually create the Nulls. I assumed they were created automatically. Is the scaling of the Body-top, Floor and BG causing an issue?

**Comment by Inventsable** [comment]
I don't think that's directly involved but it led me to assume that you had it at an initial size, parented the Rubberhoses to something, scaled those assets up as well as the Null parent, then deleted the parent, because when I try to "reverse" this process by parenting the Rubberhoses and scaling up ~250%, they appear to be the around the correct size on render if I slide their X position.

You didn't parent or adjust them in any way? When I create basehoses and drag the controllers into position, then adjust the hose weight (taking care not to drag the hose, only dragging the controllers), I get results as expected:

<img width="1052" alt="ice_screenshot_20190618-230354" src="https://user-images.githubusercontent.com/37279677/59740697-b92dbc80-921d-11e9-92a6-63e85fcad945.png">

**Comment by grodasy10** [comment]
I didn't parent, or adjust them in any way. You're right though, I just tried creating a new hose then copying all of the position frames onto the new one and it works perfectly! However if I duplicate or make any changes to the current hoses, they all end up pinned to the top left of the screen.
Thanks for your help on this, I'm not sure why it's happened but at least I know how to move forward.

**Comment by Inventsable** [comment]
Very strange indeed! At least the keys can be saved and you wouldn't lose much time.

Can we close this issue? I'm not sure what happened, but I feel it's a problem on the AE/Rubberhose side rather than Lottie (despite it appearing correct in AE's preview screen).

**Comment by grodasy10** [comment]
Absolutely, you're correct. Seems to be an AE/rubber hose issue. Thanks.

**Comment by bodymovin** [comment]
Hi, I found the issue.
It was an AE expression that was not being converted to valid javascript during export.
It will be fixed on the next deploy.

**Comment by kudanai** [comment]
Closing again as fixed in the updates
